tests/test_browser.py

```diff
@@ -2081,7 +2081,15 @@ def test_sdl2_key(self):
           event.initKeyEvent("keydown", true, true, window,
                              0, 0, 0, 0,
                              c, c);
-          document.dispatchEvent(event);
+          var prevented = !document.dispatchEvent(event);
+
+          //send keypress if not prevented
```

Comment:
I'm trying to emulate the behaviour that calling preventDefault on keydown events stops keypress. (The problem that https://github.com/emscripten-ports/SDL2/commit/e60718c37e5b4dbabdc87c21193ee9e6e9bf5 is working around)